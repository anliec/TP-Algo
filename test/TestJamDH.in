JAM_DH 1

ADD 3 2015 11 11 20 00 1 N
ADD 3 2015 11 11 19 00 1 R
JAM_DH 1

ADD 2 2015 11 11 18 00 1 V
ADD 2 2015 11 11 17 00 1 J
JAM_DH 1

ADD 3 2015 11 11 19 01 1 V
ADD 4 2015 11 11 20 01 1 J
ADD 5 2015 11 11 19 02 1 R
JAM_DH 1

ADD 2 2015 11 11 17 01 1 J
ADD 2 2015 11 11 17 02 1 N
JAM_DH 1

JAM_DH 2
EXIT
