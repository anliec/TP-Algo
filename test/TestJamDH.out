1 0 100%
1 1 100%
1 2 100%
1 3 100%
1 4 100%
1 5 100%
1 6 100%
1 7 100%
1 8 100%
1 9 100%
1 10 100%
1 11 100%
1 12 100%
1 13 100%
1 14 100%
1 15 100%
1 16 100%
1 17 100%
1 18 100%
1 19 100%
1 20 100%
1 21 100%
1 22 100%
1 23 100%
1 0 100%
1 1 100%
1 2 100%
1 3 100%
1 4 100%
1 5 100%
1 6 100%
1 7 100%
1 8 100%
1 9 100%
1 10 100%
1 11 100%
1 12 100%
1 13 100%
1 14 100%
1 15 100%
1 16 100%
1 17 100%
1 18 100%
1 19 0%
1 20 0%
1 21 100%
1 22 100%
1 23 100%
1 0 100%
1 1 100%
1 2 100%
1 3 100%
1 4 100%
1 5 100%
1 6 100%
1 7 100%
1 8 100%
1 9 100%
1 10 100%
1 11 100%
1 12 100%
1 13 100%
1 14 100%
1 15 100%
1 16 100%
1 17 100%
1 18 100%
1 19 0%
1 20 0%
1 21 100%
1 22 100%
1 23 100%
1 0 100%
1 1 100%
1 2 100%
1 3 100%
1 4 100%
1 5 100%
1 6 100%
1 7 100%
1 8 100%
1 9 100%
1 10 100%
1 11 100%
1 12 100%
1 13 100%
1 14 100%
1 15 100%
1 16 100%
1 17 100%
1 18 50%
1 19 0%
1 20 0%
1 21 100%
1 22 100%
1 23 100%
1 0 100%
1 1 100%
1 2 100%
1 3 100%
1 4 100%
1 5 100%
1 6 100%
1 7 100%
1 8 100%
1 9 100%
1 10 100%
1 11 100%
1 12 100%
1 13 100%
1 14 100%
1 15 100%
1 16 100%
1 17 100%
1 18 33%
1 19 0%
1 20 0%
1 21 100%
1 22 100%
1 23 100%
1 0 100%
1 1 100%
1 2 100%
1 3 100%
1 4 100%
1 5 100%
1 6 100%
1 7 100%
1 8 100%
1 9 100%
1 10 100%
1 11 100%
1 12 100%
1 13 100%
1 14 100%
1 15 100%
1 16 100%
1 17 100%
1 18 50%
1 19 0%
1 20 0%
1 21 100%
1 22 100%
1 23 100%
1 0 100%
1 1 100%
1 2 100%
1 3 100%
1 4 100%
1 5 100%
1 6 100%
1 7 100%
1 8 100%
1 9 100%
1 10 100%
1 11 100%
1 12 100%
1 13 100%
1 14 100%
1 15 100%
1 16 100%
1 17 50%
1 18 50%
1 19 0%
1 20 0%
1 21 100%
1 22 100%
1 23 100%
1 0 100%
1 1 100%
1 2 100%
1 3 100%
1 4 100%
1 5 100%
1 6 100%
1 7 100%
1 8 100%
1 9 100%
1 10 100%
1 11 100%
1 12 100%
1 13 100%
1 14 100%
1 15 100%
1 16 100%
1 17 66%
1 18 50%
1 19 0%
1 20 0%
1 21 100%
1 22 100%
1 23 100%
2 0 100%
2 1 100%
2 2 100%
2 3 100%
2 4 100%
2 5 100%
2 6 100%
2 7 100%
2 8 100%
2 9 100%
2 10 100%
2 11 100%
2 12 100%
2 13 100%
2 14 100%
2 15 100%
2 16 100%
2 17 100%
2 18 100%
2 19 100%
2 20 100%
2 21 100%
2 22 100%
2 23 100%
